pkg/csi/service/common/commonco/k8sorchestrator/k8sorchestrator.go

```diff
@@ -337,6 +339,26 @@ func Newk8sOrchestrator(ctx context.Context, controllerClusterFlavor cnstypes.Cn
 	return k8sOrchestratorInstance, nil
 }
 
+func getReleasedVanillaFSS() map[string]string {
+	return map[string]string{
+		common.CSIMigration:                   "true",
+		common.OnlineVolumeExtend:             "true",
+		common.TriggerCsiFullSync:             "false",
```

Comment:
Can we remove `TriggerCsiFullSync` from this list? FVT would want to toggle this FSS and run tests. IsFSSEnabled needs to read this value from configmap.

Reply:
Removed.